setup: add bits for setuptools-61

Quoting https://github.com/pypa/setuptools/issues/3227
"Setuptools >= 61, intentionally changes the way packages are built in the
 sensec that it will try to find files and fail if something is weird.

 Empty packages (like this one), are asked to explicitly add packages=[]
 to their configuration.

 This intentional change in behaviour is described in
 https://setuptools.pypa.io/en/latest/history.html#v61-0-0."

https://bugs.gentoo.org/836645
https://github.com/virt-manager/virt-manager/commit/46dc0616308a73d1ce3ccc6d716cf8bbcaac6474
--- setup.py.orig
+++ setup.py
@@ -547,4 +547,5 @@ def run(self):
     },
 
     distclass=VMMDistribution,
+    packages=[],
 )
